data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -64,6 +71,13 @@ class ExtractTransformLoad:
     # COLUMNS_TO_KEEP is used to identify which columns to keep in the output df.
     COLUMNS_TO_KEEP: typing.List[str] = None
 
+    # INPUT_GEOID_TRACT_FIELD_NAME is the field name that identifies the Census Tract ID
+    # on the input file
+    INPUT_GEOID_TRACT_FIELD_NAME: str = None
+
+    # NULL_REPRESENTATION is how nulls are represented on the input field
+    NULL_REPRESENTATION: str = None
```

Comment:
We definitely should use a null representation (think about how much easier it would be for census data, which uses `-666666666` or something) and that representation is specified in some, not all, ETLs. This is to say -- I think it makes sense to have but should be an optional argument.

Reply:
Makes sense to make it a list.

Reply:
Got it, so we'll make this a list?